src/components/list-option/list-option.test.jsx

```diff
@@ -1,39 +1,96 @@
-import React from 'react';
+import React, {
+  createRef,
+} from 'react';
 import renderer from 'react-test-renderer';
-import { render, screen } from '@testing-library/react';
+import {
+  act,
+  render,
+  screen,
+} from '@testing-library/react';
 import ListOption from './list-option.jsx';
 
 describe('src/components/list-option/list-option', () => {
+  const requiredProps = {
+    children: 'Test option',
+  };
+
   it('renders defaults', () => {
-    const tree = renderer.create(<ListOption />).toJSON();
+    const tree = renderer.create(<ListOption {...requiredProps} />).toJSON();
     expect(tree).toMatchSnapshot();
   });
 
+  describe('children API', () => {
+    it('accepts a node as children', () => {
+      render(<ListOption><div><span>Text is here!</span></div></ListOption>);
+      expect(screen.getByText('Text is here!')).toBeInTheDocument();
+    });
+  });
+
+  describe('defaultActive API', () => {
+    it('can be set', () => {
+      render(<ListOption {...requiredProps} defaultActive={true} />);
+      expect(screen.getByText('Test option')).toHaveAttribute('tabindex', '0');
+    });
+
+    it('can be unset', () => {
+      render(<ListOption {...requiredProps} defaultActive={false} />);
+      expect(screen.getByText('Test option')).toHaveAttribute('tabindex', '-1');
+    });
+  });
+
   describe('selected API', () => {
     it('is selected when selected is true', () => {
-      render(<ListOption selected>SelectMe!</ListOption>);
-      expect(screen.getByText('SelectMe!')).toHaveAttribute('aria-selected', 'true');
+      render(<ListOption {...requiredProps} selected={true} />);
```

Comment:
If I'm not mistaken, boolean props don't need to have `true` be assigned to them.

```suggestion
      render(<ListOption {...requiredProps} selected />);
```

Reply:
Yeah, I've been wavering on this but this is my latest train of thought:

Since we're testing `true` and `false`, I kinda want it to be explicit in both cases.

For example, in the `false` case, the "JSX way" of doing things is by not specifying the prop. But I want the reader to understand that the test is actually setting `selected` to `false.

In the same vein, I want to show the reader that I am testing `selected` set to `true`.